repo/packages/N/neo4j-replica/1/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "https://s3.amazonaws.com/downloads.mesosphere.io/universe/assets/icon-service-neo4j-small.png",
+    "icon-medium": "https://s3.amazonaws.com/downloads.mesosphere.io/universe/assets/icon-service-neo4j-medium.png",
+    "icon-large": "https://s3.amazonaws.com/downloads.mesosphere.io/universe/assets/icon-service-neo4j-large.png"
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "dcos-neo4j": "neo4j/neo4j-dcos:1.0.0-3.1.1"
```

Comment:
Does it make sense to have the same version schema for the docker image than for the package version above?